### DISCUSSION TITLE
Treeview does not appear in the latest update

### DISCUSSION THREAD

**Discussion by ntahlahweh** [TomSchimansky/CustomTkinter#1714] [Q&A] [answered]
Hello,

I just updated my customtkinter to the latest one which is version 5.1.3. However when I run code I found out that the Treeview does not appear. Previously I used 5.0.3 (sorry for I did not update it for a long time), it runs accordingly but now it does not. I have look around for answer but I did not find any. 

Can anyone please assist me. Here are the snippet of the code. Also attached is how it looks like when I run it. 

Thank you

```python
order = True
cwd = os.path.dirname(__file__) #get current directory 
up_dir =  os.path.abspath(os.path.join(cwd, "analysis"))
df = pd.read_csv(up_dir + "\\cb_analysis.csv")  # create dataframe
newdf = 0 # just to declare newdf variable
user_choice = []
style = ttk.Style(root)
style.theme_use('clam')

frame_tree = customtkinter.CTkFrame(master=tab_2, width=1350, height=385)
frame_tree.grid(row=0, column=0, columnspan=6, pady=20, padx=20, sticky="nsew")
frame_tree.place(relx=0.01, rely=0.03)

tree = ttk.Treeview(master=frame_tree, selectmode="browse", show="headings", height=10, columns=("Combiner Box", "Date", "Month", "Year", "Time"))  #create treeview and fill data
tree.pack(fill="both", expand=True)
vs = ttk.Scrollbar(master=tree,orient="vertical", command=tree.yview)# create Vertical Scrollbar
tree.configure(yscrollcommand=vs.set)  # connect to Treeview
vs.pack(side=RIGHT, fill=Y)

def my_disp():  # function to read and loop cb_analysis
    global df  # use the global df variable
    # Reload the CSV file to update the dataframe
    df = pd.read_csv(up_dir + "\\cb_analysis.csv")
    # Clear the treeview
    tree.delete(*tree.get_children())

    l1 = list(df)  # List of column names as headers
    r_set = df.to_numpy().tolist()  # Create list of list using rows

    for col in l1:
        tree.column(col, width=100, anchor="c")
        tree.heading(col, text=col)

    # Adding data to treeview
    for dt in r_set:
        v = [r for r in dt]  # creating a list from each row
        tree.insert("", "end", values=v)
    
    # Update filter_cb combobox
    filter_cb.configure(values=['All'] + list(df["Combiner Box"].sort_values().unique().astype(str)))
    # Update filter_month combobox
    filter_month.configure(values=['All'] + list(df["Month"].sort_values().unique().astype(str)))
    # Update filter_year combobox
    filter_year.configure(values=['All'] + list(df["Year"].sort_values().unique().astype(str)))
```

![image](https://github.com/TomSchimansky/CustomTkinter/assets/48423509/14998ff9-869b-4fdf-b20e-892a7366b2b5)

**Comment by ghost** [comment]
This isn't CustomTkinter, it's `tkinter.ttk`. You're having issues with the treeview, which, unfortunately, is not a CustomTkinter widget.

**Reply by ntahlahweh** [reply, marked as answer]
since you mentioned that... I found out that the problem why the treeview does not appear is because of the 
```python
frame_tree = customtkinter.CTkFrame(master=tab_2, width=1350, height=385)
frame_tree.grid(row=0, column=0, columnspan=6, pady=20, padx=20, sticky="nsew")
frame_tree.place(relx=0.01, rely=0.03)
```


I don't know why it is happening so I change it to:  

```python
frame_tree = Frame(master=tab_2)
frame_tree.grid(row=0, column=0, columnspan=6, pady=20, padx=20, sticky="nsew")
frame_tree.place(relx=0.01, rely=0.03, width=1350, height=385)
```

and its working! 

![image](https://github.com/TomSchimansky/CustomTkinter/assets/48423509/d187e129-137b-45d5-8930-f697c13552c4)

**Reply by avalon60** [reply]
Looks like a CustomTkinter bug.

**Reply by ghost** [reply]
Oh, huh yeah it is a CTk bug. Maybe the sizing of the CTkFrame meant that there wasn't enough space for the Treeview? That's my best guess so far.

**Comment by ghost** [comment]
Looking at the code, from what I understand, the only way your `Treeview` widget can be empty is if the iterable `dt` is empty. I'm assuming `dt` exists and is not a typo for `df`.

**Comment by ElectricCandlelight** [comment]
Why are you doing free_tree.grid() and then .place()

You should only be using one layout manger per container.

**Reply by avalon60** [reply]
I agree with @ElectricCandlelight,  and using the place method is not advised, as it can complicate matters if you need to start introducing new widgets,

**Reply by ntahlahweh** [reply]
I am still learning tkinter and python, thank you for the advice though! I've already make some changes based on your advice
